.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,10 @@
+name: "Validate Gradle Wrapper"
+on: [push]
```

Comment:
Should not this be: `on: [push, pull_request]`?

Reply:
Seems like it since it was also fixed over at RxJava: https://github.com/ReactiveX/RxJava/pull/6849